app/src/main/java/org/fossasia/openevent/general/di/Modules.kt

```diff
@@ -259,7 +258,7 @@ val networkModule = module {
         val builder = OkHttpClient().newBuilder()
             .connectTimeout(connectTimeout.toLong(), TimeUnit.SECONDS)
             .readTimeout(readTimeout.toLong(), TimeUnit.SECONDS)
-            .addInterceptor(get())
+            .addInterceptor(RequestAuthenticator(get()))
```

Comment:
Why not just get()?

Reply:
Displaying error-
```
@JvmName public final inline fun addInterceptor(crossinline block: (chain: Interceptor.Chain) -> Response): OkHttpClient.Builder defined in okhttp3.OkHttpClient.Builder
public final fun addInterceptor(interceptor: Interceptor): OkHttpClient.Builder defined in okhttp3.OkHttpClient.Builder
```

Reply:
You are passing `RequestAuthenticator(get())` and `single { RequestAuthenticator(get()) }` is already defined above.  Either remove that or use get

Reply:
Updated